static/main.js

```diff
@@ -349,6 +354,9 @@ require("monaco-loader")().then(function () {
         setupAdd($('#add-editor'), function () {
             return Components.getEditor();
         });
+        setupAdd($('#add-compiler'), function () {
+            return Components.getEditor();
```

Comment:
This is a duplicate of the function directly above, and by the looks of it, it seems like it should not exist, as there would be ambiguous as to which editor the new compiler should belong to

Reply:
Sounds good.